pkg/containerd/events.go

```diff
@@ -287,6 +401,45 @@ func createPortMappingFromString(portMapping string) (nat.PortMap, error) {
 	return portMap, nil
 }
 
+func extractIPAddress(line string) (string, error) {
```

Comment:
This could use a doc comment on what this does. Or maybe move running `nsenter ip` into this function.